pyro/distributions/zero_inflated.py

```diff
@@ -111,10 +127,11 @@ class ZeroInflatedNegativeBinomial(ZeroInflatedDistribution):
     arg_constraints = {"gate": constraints.unit_interval,
                        "total_count": constraints.greater_than_eq(0),
                        "probs": constraints.half_open_interval(0., 1.),
-                       "logits": constraints.real}
+                       "logits": constraints.real,
```

Comment:
not sure what the best naming convention is for ZINB. there are two types of probabilities, which muddies the watter

Reply:
yeah. I guess `gate_logits` makes the most sense. WDYT of using that for other distributions too, for consistency?

Reply:
but then do i also change `probs` -> `gate_probs`?

Reply:
no, the pairs are `(gate, gate_logits)` and `(probs, logits)`. We shouldn't rename any existing parameters.